src/privileged/autoplay/api.js

```diff
@@ -0,0 +1,397 @@
+"use strict";
+
+ChromeUtils.import("resource:///modules/PermissionUI.jsm");
+ChromeUtils.import("resource:///modules/SitePermissions.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionCommon.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionStorage.jsm");
+ChromeUtils.import("resource://gre/modules/PopupNotifications.jsm");
```

Comment:
I suspect you could make more of these lazy - basically anything that has the possibility of not being used is good to not directly import.

Reply:
Ok, thank you!